llvm/test/CodeGen/SPIRV/transcoding/OpBitReverse_i32.ll

```diff
@@ -1,13 +1,14 @@
-; RUN: llc -O0 -mtriple=spirv32-unknown-unknown %s -o - | FileCheck %s --check-prefix=CHECK-SPIRV
+; RUN: llc -O0 -mtriple=spirv32-unknown-vulkan %s -o - | FileCheck %s --check-prefix=CHECK-SPIRV
```

Comment:
Don't think this is correct. `SPV_KHR_bit_instructions` can be used to allow OpBitReverse in Kernel.
I'd suggest adding 2 lines:
- `-mtriple=spirv-unknown-vulkan`
- `-mtriple=spirv32-unknown-unknown` with an XFAIL + github issue as it seems the extension is not emitted, hence shader capability is wrongly added.